### PR TITLE
fix(messaging): iOS - fix bug where tapping a notification when the app is either in the background or closed, would not trigger any listener

### PR DESCRIPTION
**Pull request by jaco-terbraak**
…ackground or closed did not trigger any function call

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ x ] The changes have been tested successfully.
- [ x ] A changeset has been created (`npm run changeset`).

### CODE REVIEW

File: packages/messaging/ios/Plugin/FirebaseMessaging.swift [hunk 1]
```diff
@@ -119,6 +121,25 @@ import FirebaseCore
...
+    }
+
+    public func userNotificationCenter(
+        _ center: UNUserNotificationCenter,
+        didReceive response: UNNotificationResponse,
+        withCompletionHandler completionHandler: @escaping () -> Void
+    ) {
+        self.handleNotificationActionPerformed(response: response)
+        completionHandler()
+    }
```
Comment (robingenz) [inline packages/messaging/ios/Plugin/FirebaseMessaging.swift:142]: Please move these public methods above the two private methods.

**Comment by jaco-terbraak** [comment]
hold on, it's not completely working like it should.

**Review by robingenz — CHANGES_REQUESTED** [review]
Thank you for your contribution! I will test this in the next week.

**Comment by robingenz** [comment]
@jaco-terbraak Okay, you were faster. ^^

**Pull request closed without merging**